mojaloop-simulator/values.yaml

```diff
@@ -503,7 +503,7 @@ defaults: &defaults
         # state you can use a preexisting file. If running in a container, you can mount a sqlite file as a
         # volume in the container to preserve state between runs.
         # Use MODEL_DATABASE: :memory: for an ephemeral in-memory database
-        MODEL_DATABASE: ./model.sqlite
+        MODEL_DATABASE: ':memory:'
```

Comment:
`./model.sqlite` seems to be somehow generated when running the application but during the upgrade this no longer seems to be generated. I can't find anything in the sqlite packages noting a change. It should be fine to use memory here I believe.